scripts/MainMenu/button.js

```diff
@@ -7,13 +17,14 @@ function Button(x, y, width, height, fontSize, message, onClick, scene) {
   this.message = message;
   this.onClick = onClick || function () {};
   this.scene = scene || null;
+  this.borderRadius = borderRadius || null;
```

Comment:
Border radius should = borderRadius || 0
That way it doesn't crash if not specified.